[docs] Gh #5196 describe current limitations (#5428)
DevExpress/testcafe: docs/articles/documentation/guides/basic-guides/obtain-client-side-info.md

This approach only works for Node.js callbacks that fire during a test run. To ensure that the test function
does not finish before the callback, suspend the test until the callback fires. You can introduce a promise and synchronously wait for it to complete, as shown in the example above.

> > The `boundTestRun` option requires the same test controller instance that is passed to the function used in a test declaration. It's unable to work with imported test controllers.

Reviewer: ```suggestion
> The `boundTestRun` option requires the same test controller instance that is passed to the function used in the test declaration. It cannot work with imported test controllers.
```

>

> > The `boundTestRun` option is unable to pass a test controller instance to a function declared in another module or class.

Reviewer 2: Same as below

Author: edited (removed redundant second sentence)

> > For more info on imported test controllers, refer to the [Implicit Test Controller Use](../../reference/test-api/testcontroller/README.md#implicit-test-controller-use) section.

Reviewer: ```suggestion
> For more information on imported test controllers, refer to the following section: [Implicit Test Controller Use](../../reference/test-api/testcontroller/README.md#implicit-test-controller-use).
```

This approach only works for Node.js callbacks that fire during a test run. To ensure that the test function
does not finish before the callback, suspend the test until the callback fires. You can introduce a promise and synchronously wait for it to complete, as shown in the example above.

> > The `boundTestRun` option requires the same test controller instance that is passed to the function used in a test declaration. It's unable to work with imported test controllers.

Reviewer 3: It's a difficult to understand without examples.

Reviewer 4: I think that adding code to a note is too much, but a link to the imported test controller docs could work well

Author: done
